src/main/java/net/glowstone/GlowWorld.java

```diff
@@ -1790,6 +1790,15 @@ private void pulseTickMap() {
         return tickMap;
     }
 
+    public void requestPulse(GlowBlock block) {
+        ItemTable itemTable = ItemTable.instance();
+        BlockType type = itemTable.getBlock(block.getType());
+        if (type == null || type.getPulseTickSpeed() == null) {
```

Comment:
I meant in pulseTickMap(), so that we don't have to store those values any more. So less memory.